MockKit/MockCGMManager.swift

```diff
@@ -457,11 +572,17 @@ extension MockCGMState: CustomDebugStringConvertible {
         ## MockCGMState
         * isStateValid: \(isStateValid)
         * trendType: \(trendType as Any)
+        * urgentLowGlucoseThresholdValue: \(urgentLowGlucoseThresholdValue)
         * lowGlucoseThresholdValue: \(lowGlucoseThresholdValue)
         * highGlucoseThresholdValue: \(highGlucoseThresholdValue)
+        * cgmLowerLimitValue: \(cgmLowerLimitValue)
+        * cgmUpperLimitValue: \(cgmUpperLimitValue)
+        * highGlucoseThresholdValue: \(highGlucoseThresholdValue)
         * glucoseValueType: \(glucoseValueType as Any)
         * cgmStatusHighlight: \(cgmStatusHighlight as Any)
-        * cgmStatusProgress: \(cgmStatusProgress as Any)
+        * cgmLifecycleProgress: \(cgmLifecycleProgress as Any)
```

Comment:
[Question] Why is the `as Any` needed here? What does that do (as opposed to just `\(cgmLifecycleProgress)`)? (I actually don't know so am asking.)

Reply:
As I understand it, it removes the warning when making a string from an optional. Similar to `String(describing:)`